lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -29,8 +29,10 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
     return {
       category: 'Performance',
       name: 'external-anchors-use-rel-noopener',
-      description: 'Site opens external anchors using rel="noopener".',
-      helpText: 'Anchors that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener" target="_blank">opening page\'s performance does not suffer.</a>',
+      description: 'Site opens external anchors using rel="noopener"',
+      helpText: 'Open new tabs using <code>rel="noopener"</code> to improve performance and ' +
```

Comment:
> Not sure if our linter will complain though....

we have `ignoreUrls` turned on for our max line length check, so any line with a URL in it won't fail the lint check.